lib/allocate.rs

```diff
@@ -3,11 +3,11 @@ use std::collections::BTreeSet;
 use std::ops;
 
 pub fn allocate<B: Ord + Clone, N: num::Num + ops::Add + cmp::Ord + Copy>(
-    allocated: &crate::ddlog_std::Set<N>,
-    toallocate: &crate::ddlog_std::Vec<B>,
+    allocated: &ddlog_std::Set<N>,
```

Comment:
Thanks for reviewing, this is really valuable!

All Rust code, including libraries used to live in the same namespace.  Fortunately, this is no longer the case, so yes, at least as far as native library code goes, imports should work. I sometimes prefer qualified names, especially when the actual type or function name is too generic.  I noticed that you feel different about these things :)

I'll fix this library as you suggest.  Overall, I feel that our native library code has been tortured to death by all the refactorings and looks terrible in many ways, including those identified in your review.

Reply:
I generally prefer importing things since it becomes nicer to work within, for overtly generic names (such as `Vec`) I'd just alias it to `DDlogVec` or something

Reply:
Heh, we can agree to disagree.  In case of our native libraries, though, any consistent style would be an improvement :)